tiles/Make.inc

```diff
@@ -17,6 +17,13 @@ TILESSRC:=\
 
 CLEAN+=$(TILESSRC:.cc=.o)
 
+OS := $(shell uname)
```

Comment:
I also added a branch to revert to make static library on other environments than OSX.